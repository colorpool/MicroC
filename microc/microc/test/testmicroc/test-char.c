void main(){
  char a;
  a='b';
  print 'c' : a;
}