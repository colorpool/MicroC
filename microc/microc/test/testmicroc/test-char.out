24 19 0 5 25 15 1 13 0 0 1 27 98 12 15 -1 13 0 0 1 11 23 15 -1 15 -1 21 -1